### ISSUE TITLE
Seed Everywhere not working after updating ComfyUI

### ISSUE THREAD

**Issue by memedoggo** [chrisgoringe/cg-use-everywhere#237]
![zzzxzxz](https://github.com/user-attachments/assets/a0cf25e4-74cc-46a1-bdfd-ad1e63d1b357)

Used to be just Scheduler but I converted that to widget and it worked
After the latest ComfyUI update, Noise Seed also don't work and I have to convert it to widget...

`got prompt
Failed to validate prompt for output 473:
* ImpactKSamplerAdvancedBasicPipe 319:
  - Required input is missing: cfg
  - Return type mismatch between linked nodes: scheduler, ['normal', 'karras', 'exponential', 'sgm_uniform', 'simple', 'ddim_uniform', 'beta'] != ['normal', 'karras', 'exponential', 'sgm_uniform', 'simple', 'ddim_uniform', 'beta', 'AYS SDXL', 'AYS SD1', 'AYS SVD', 'GITS[coeff=1.2]']
Output will be ignored
model weight dtype torch.float16, manual cast: None
model_type EPS
Using pytorch attention in VAE
Using pytorch attention in VAE
Requested to load SDXLClipModel
Loading 1 new model
loaded completely 0.0 1560.802734375 True
D:\ComfyUI\ComfyUI\comfy\ldm\modules\attention.py:407: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:263.)
  out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)
Prompt executed in 19.13 seconds
got prompt
Failed to validate prompt for output 473:`

**Comment by JorgeR81** [comment]
Try updating.

It's working for me.

Also with that sampler.

![pipe](https://github.com/user-attachments/assets/a89d7f82-a2c9-48da-9ed9-bb020de8b228)

**Comment by chrisgoringe** [comment]
The error message says cfg is missing

**Comment by JorgeR81** [comment]
Now that I'm looking better at it, the Sampler in the image does not have a `cfg` widget / input !

But my Sampler has it.

Maybe it was an `Impact-Pack` bug that got fixed.

![ip](https://github.com/user-attachments/assets/4a65714e-7809-4f72-91f0-91bfa9b8a3ff)

**Comment by chrisgoringe** [comment]
I think this is working on the 1.16 branch - see #287 option 2

**Comment by chrisgoringe** [comment]
Update to UE nodes 6.0.0 which I believe to be working.
